aries-site/src/data/structures/learn.js

```diff
@@ -16,6 +16,17 @@ export const learn = [
     relatedContent: [],
     tags: [],
   },
+  {
+    name: 'The Box Model Part One',
+    render: 'The Box Model: Part One',
+    description:
+      'Build a hero section to learn about fundamental Box properties.',
+    seoDescription:
+      'Build a hero section to learn about fundamental Box properties.',
+    sections: [],
+    relatedContent: [],
```

Comment:
Since we are using `Box` `Heading` `Image` in the example and `Text`,`Button` should we include these in related content?